packages/babel-parser/src/tokenizer/state.ts

```diff
@@ -152,20 +279,36 @@ export default class State {
   }
 
   clone(skipArrays?: boolean): State {
```

Comment:
Actually we don't use `skipArrays` anywhere. :)

Reply:
Can we just remove it then? :)